src/components/PageFooter.tsx

```diff
@@ -123,21 +141,39 @@ Props) => {
                 justifyContent: 'space-between',
                 width: { _: '100%', md: '50%' },
                 ...footerLinks,
+                ...gridLayout,
               }}
             >
               <Link href={routes.twitterUrl} muiProps={{ target: '_blank' }}>
-                Twitter
+                <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                  <XIcon color="inherit" fontSize="small" sx={{ mr: 0.75 }} />
+                  Twitter
+                </Box>
               </Link>
               <Link href={routes.youtubeUrl} muiProps={{ target: '_blank' }}>
-                Youtube
+                <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                  <LinkedInIcon color="inherit" fontSize="small" sx={{ mr: 0.75 }} />
```

Comment:
this doesn't look right - the youtube link has a linkedin icon.  confirmed on the crossplane.io site after this was deployed.

Can we please fix this?
![Screenshot 2024-02-02 at 11 47 42 AM](https://github.com/crossplane/website/assets/4313439/c9f899d5-7378-424d-b4de-bbc304ffbe1e)

Reply:
PR with the fix: https://github.com/crossplane/website/pull/56